content/3_Silver/Two_Pointers.mdx

```diff
@@ -44,15 +44,145 @@ of an interval. It can also be used to track two values in an array as shown in
 
 We want to find the longest contiguous segment of books that can be read within $t$ minutes. 
 
-To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ as the start and end of the segment respectively and use two pointers to track the interval.
+To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ to **represent** the beginning and end of the **segment**. However, both will start at the beginning of the array. These numbers can be thought of as pointers, hence the name "two pointers".
 
 For every $\texttt{left}$ index, we need to increase our $\texttt{right}$ pointer until the total time for that segment is the maximum while being less than $t$.
 
 We will keep track of the max segment size with $\texttt{ans}$, and take the maximum $\texttt{right} - \texttt{left}$ value.
 
-We can then continually increment $\texttt{left}$ by 1.
-
-**The $\texttt{right}$ pointer does not restart after an iteration, it simply keeps moving right; therefore, since both pointers will move at most $N$ times, the overall time complexity is $\mathcal{O}(N)$.**
+We can then continually increment $\texttt{left}$ by 1. After incrementing the left index, the time used decreases, hence the right pointer never has to move back (left). Thus:
+
+**Since both pointers will move at most $N$ times, the overall time complexity is $\mathcal{O}(N)$.**
```

Comment:
Don't know if this needs to be bolded either

Reply:
That was bolded before my commits

Reply:
Yeah I just don't know what the purpose the holding serves. Time complexity analysis isn't normally bolded in any other solution/explanation. But yeah it's ultimately up to ur discretion